app/components/Notification/index.jsx

```diff
@@ -33,7 +33,7 @@ let NotifiItem = React.createClass({
       color: this.props.read ? "#888" : "#000"
     };
     return (
-      <Layout horizontal>
+      <Layout horizontal className="mui-menu-item">
```

Comment:
@zycbobby Assign `mui-menu-item` to your items manually. No need to change other code.